### ISSUE TITLE
Error api.graphql.imdb.com

### ISSUE THREAD

**Issue by guardiaopt** [tboothman/imdbphp#329]
Today the script stopped working, does anyone else have the same problem?

Caught exception: Failed fetch url [https://api.graphql.imdb.com/]

**Comment by ykdsoft** [comment]
yes. i have the same problem. graphql api requests not working. also i have a problem plot_split, plotoutline and synopsis methods since yesterday.
